src/main/java/net/snowflake/client/core/SfSqlArray.java

```diff
@@ -74,4 +80,19 @@ public ResultSet getResultSet(long index, int count, Map<String, Class<?>> map)
 
   @Override
   public void free() throws SQLException {}
+
+  public String getJsonString() {
+    try {
+      return SnowflakeUtil.mapJson(elements);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
```

Comment:
Is it a proper wrapping? I'm not entierely sure if it is ok or not.

Reply:
Fixed